app/services/make_an_offer.rb

```diff
@@ -0,0 +1,16 @@
+class MakeAnOffer
+  Response = Struct.new(:successful?, :application_choice)
+
+  def initialize(application_choice:, offer_conditions:)
+    @application_choice = application_choice
+    @offer_conditions = offer_conditions
+  end
+
+  def call
+    @application_choice.status = @offer_conditions.present? ? :conditional_offer : :unconditional_offer
+    @application_choice.offer = @offer_conditions.present? ? @offer_conditions : { 'conditions' => [] }
+    @application_choice.save
+    offer_response = Response.new(true, @application_choice)
+    offer_response
```

Comment:
`Response.new(true, @application_choice)` is enough

Reply:
👍